src/memote/suite/tests/test_consistency.py

```diff
@@ -64,6 +64,44 @@ def test_stoichiometric_consistency(model):
     assert is_consistent, ann["message"]
 
 
+@annotate(title="Inconsistent Minimal Sets", format_type="count")
```

Comment:
We decided to merge this test case with the above. For that, change the `format_type` to `"percent"`. This should display the metric in the report (please check that it displays correctly).

The test data should then be changed as follows:
```python
ann["data"] = {
    "unconserved_metabolites": [...],
    "inconsistent_minimal_stoichiometries": [...]
}
ann["metric"] = len(ann["data"]["unconserved_metabolites"]) / len(model.metabolites)
```